user-guide/Advanced_Functionality/DataMiner_Agents/Configuring_a_DMA/Setting_up_HTTPS_on_a_DMA.md

```diff
@@ -62,8 +62,8 @@ To do so:
 
       This name **should also be configured in the DNS server** pointing to the IP address of the DMA, so that the DMA can be reached using the configured name.
 
-      > [!WARNING]
-      > Do not use wildcard configurations if you want to use your DataMiner Agent to connect your system to dataminer.services, as this is not supported. In that case, use the FQDN (e.g. "dma01.skyline.be") instead.
+      > [!IMPORTANT]
```

Comment:
FYI, I've also changed the alert level to important, as this is essential information for user success, rather than an action with dangerous consequences.